internal/logging/logging.go

```diff
@@ -43,11 +42,11 @@ func initLogger() {
 	if cfg.CwAccessKey != "" {
 		cred := credentials.NewStaticCredentials(cfg.CwAccessKey, cfg.CwSecretKey, "")
 		awsconf := aws.NewConfig().WithRegion(cfg.CwRegion).WithCredentials(cred)
-		hook, err := lc.NewBatchingHook(cfg.CwLogGroup, cfg.CwLogStream, awsconf, 10*time.Second)
+		hook, err := lc.NewBatchWriter(cfg.CwLogGroup, cfg.CwLogStream, awsconf)
```

Comment:
Change required due to changes to the latest version of the `platform-go-middlewares`